circuits/modulus.circom

```diff
@@ -66,44 +67,82 @@ template ModulusWith25519(n) {
   component mod;
   component adder;
   component mod2pfinal;
+
   if (n < 255) {
-    for (i=0; i<n; i++) {
-      out[i] <== a[i];
+    /*
+     * Easy case when the number is less than 255 bits.
+     * In this case modulus over 2^255 - 19 would be
```

Comment:
"would be the same number" or "would result in the same number"

Reply:
Fixed in the new revision.